app/src/app/pages/artist/artist.component.ts

```diff
@@ -40,12 +40,23 @@ export class ArtistComponent implements OnInit, OnDestroy {
       const artistId = params.get('artistId');
       /** Use data service to fetch entity from database */
       this.artist = await this.dataService.findById<Artist>(artistId, EntityType.ARTIST);
+      
+      this.artist.birth = this.formatDate(this.artist.birth);
+      this.artist.death = this.formatDate(this.artist.death);
 
       /** load slider items */
       this.dataService.findArtworksByType(EntityType.ARTIST, [this.artist.id]).then(artworks => (this.sliderItems = shuffle(artworks)));
     });
   }
 
+  formatDate(date: string) {
```

Comment:
this might not work for formats like 'um 1920/1930', where `datePart[2]` is undefined. For the display of dateOfBirth/Death, we dont need to format the date.
For the timeline portraits, we no longer need the artists birth/death because we have no movement pages, where the artist originally was displayed in the timeline. I think we dont need this formatting function for now